less/base.less

```diff
@@ -122,7 +122,7 @@ h3 {
 }
 
 h2, h3 {
-  margin: 2rem 0 .8rem;
+  margin: 3rem 0 .8rem;
```

Comment:
Should these be in a separate PR? 

Feels a bit odd to have styling coupled with all of the content 😁

Reply:
Sure! I'll do that

Reply:
I added a PR for these changes #298